aws/cloudformation/components/www_redirect.yml.erb

```
@@ -93,7 +93,11 @@
             const querystring = request.querystring;
 
             // Start building new url
+            <%- if rack_env?(:production) -%>
+            let newUrl = 'https://<%=CDO.pegasus_hostname%>'
+            <%- else -%>
             let newUrl = 'https://<%= subdomain %>';
+            <%- end -%>
```

I can't decide if this DRYer approach is more readable or not, with the ternary:

```suggestion
            let newUrl = 'https://<%= rack_env?(:production) ? CDO.pegasus_hostname : subdomain %>';
```

I think what I'd really like to see is a helper method of some kind we can call to get the canonical hostname for the environment we're in, so we could just do:

```
            let newUrl = 'https://<%= CDO.canonical_hostname_or_whatever %>';
```

But I'm not sure how annoying that would be to do in this part of our architecture.

---

Yeah, I was considering the ternary. I'll take a look and make sure we don't already have a canonical hostname available.

---

perhaps `CDO.pegasus_hostname` is already it, and I don't need to use `subdomain`

---

in lower environments, `subdomain == CDO.pegasus_hostname`, but in prod they differ. This lets me get rid of the conditional entirely.